media-libs/libarib25/libarib25-0.2.5.20180719.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Implementation of ARIB STD-B25"
+HOMEPAGE="https://github.com/stz2012/libarib25"
+
+MY_PV="${PV%.*}-${PV#*.*.*.}"
```

Comment:
Nope, this is unreadable. Use `ver_rs` from EAPI 7 or eapi7-ver. Also, please move it above DESCRIPTION to avoid breaking the natural order of variables™.